libs/auth-api-lib/src/lib/idp-provider/models/idp-provider.model.ts

```diff
@@ -37,7 +37,7 @@ export class IdpProvider extends Model {
   helptext!: string
 
   @Column({
-    type: DataType.NUMBER,
+    type: DataType.INTEGER,
```

Comment:
No migration required?

Reply:
These decorator types are just used in jest tests and were out of sync with the postgres db schema. 👍